src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging.Extraction.csproj

```diff
@@ -0,0 +1,360 @@
+    <Folder Include="NuGet.Packaging\Definitions" />
+    <Folder Include="NuGet.Packaging\Signing\Package" />
+    <Folder Include="NuGet.Packaging\Signing\Utility" />
+    <Folder Include="NuGet.Packaging\Core\comparers" />
+    <Folder Include="NuGet.Packaging\Signing\Signatures" />
+    <Folder Include="NuGet.Packaging\Signing\Content" />
+    <Folder Include="NuGet.Packaging\Exceptions" />
+    <Folder Include="NuGet.Packaging\PackageCreation\Authoring" />
+    <Folder Include="NuGet.Packaging\Licenses" />
+    <Folder Include="NuGet.Packaging\Signing\Cms" />
+    <Folder Include="NuGet.Packaging\Signing\Timestamp" />
+    <Folder Include="NuGet.Packaging\PackageCreation\Extensions" />
+    <Folder Include="NuGet.Packaging\Signing\Archive" />
+    <Folder Include="NuGet.Packaging\Signing\DerEncoding" />
+    <Folder Include="NuGet.Packaging\Signing\TrustedSigners" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(IsCore)' == 'true' ">
+    <PackageReference Include="System.Security.Cryptography.Pkcs" />
+    <PackageReference Include="System.Security.Cryptography.Cng" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.IO.Compression" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="NuGet.Packaging\Signing\DerEncoding\SR.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>SR.resx</DependentUpon>
+    </Compile>
+    <Compile Update="Strings.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Strings.resx</DependentUpon>
+    </Compile>
+    <Compile Update="Rules\AnalysisResources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>AnalysisResources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="..\NuGet.Packaging\Rules\AnalysisResources.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>AnalysisResources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+    <EmbeddedResource Update="..\NuGet.Packaging\Signing\DerEncoding\SR.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>SR.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+    <EmbeddedResource Update="..\NuGet.Packaging\Strings.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Strings.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+    <EmbeddedResource Update="..\NuGet.Packaging\PackageCreation\Resources\NuGetResources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>NuGetResources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+
+
+  <Import Project="$(BuildCommonDirectory)common.targets" />
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+</Project>
+
```

Comment:
Can we remove empty lines?

Reply:
wow, that was a lot of extra lines, just in case they didn't see the first one. fixed in next iteration.